motr/m0crate/crate_client.h

```diff
@@ -54,6 +54,7 @@ struct crate_conf {
 	int max_rpc_msg_size;
         int col_family;
 	int log_level;
+	uint64_t addb_size;
```

Comment:
Please fix the indentation of this struct:
* use tabs to indent.

Reply:
Done. c9a1f4a